src/containerapp/azext_containerapp/containerapp_decorator.py

```diff
@@ -601,6 +617,8 @@ def construct_payload(self):
     def validate_arguments(self):
         super().validate_arguments()
         validate_create(self.get_argument_registry_identity(), self.get_argument_registry_pass(), self.get_argument_registry_user(), self.get_argument_registry_server(), self.get_argument_no_wait(), self.get_argument_source(), self.get_argument_artifact(), self.get_argument_repo(), self.get_argument_yaml(), self.get_argument_environment_type())
+        if self.get_argument_service_bindings() and len(self.get_argument_service_bindings()) > 1 and self.get_argument_customized_keys():
+            raise InvalidArgumentValueError("--bind have multiple values, but --customized-keys only can be set when --bind has single value.")
```

Comment:
1. do not have validation for update command?
2. refine the error message: e.g. --customized-keys only can be set when xxx is single